fix(root): fixes examples not opening in new window (#1419)
mi6/ic-design-system: src/content/structured/get-started/custom-theme.mdx

Most component's variants can be displayed using dark mode colours. This creates better colour contrast with backgrounds, borders and text for when components need to be used on dark backgrounds.

> These components has been given a `theme` prop, taking values of `inherit` (default), `light` and `dark`. By setting the value to either `light` or `dark` at the component level, those components will display dark mode colours, independent of any higher level themes or system settings.

Reviewer: ```suggestion
These components have been given a `theme` prop, taking values of `inherit` (default), `light` and `dark`. By setting the value to either `light` or `dark` at the component level, those components will display using the corresponding theme, independent of any higher level themes or system settings.
```
